instance/tests/models/test_utils.py

```diff
@@ -486,3 +486,35 @@ class DjangoResourceTest(SimpleResourceTestCase):
     Run the same tests as in SimpleResourceTestCase, but using DjangoResource.
     """
     make_resource = DjangoResource
+
+    def setUp(self):
+        self.make_resource.save = Mock()
```

Comment:
@itsjeyd  Oh, I forgot that DjangoResource was just defined in this file. Right. In that case, I think you could just add `save = Mock()` right in the definition of `DjangoResource`, rather than patching it or modifying it here.